js/app/historyStore.js

```diff
@@ -216,6 +207,15 @@ const HistoryStore = new Lang.Class({
 
     // Common helper functions for history stores, not for use from other
     // modules...
+    set_current_subset: function (model) {
```

Comment:
I think you amended this change into the wrong commit, squash it into the one titled "Add current-subset property to HistoryStore"